### PR TITLE
Feature/api mock

### PR DESCRIPTION
**Pull request by CebulMaciej**

### CODE REVIEW

File: AI_NETCORE_API/AI_NETCORE_API/Controllers/ValuesController.cs [hunk 1]
```diff
@@ -1,53 +1,53 @@
-﻿using System;
```
Comment (PawelBalcerek) [inline AI_NETCORE_API/AI_NETCORE_API/Controllers/ValuesController.cs:1]: Do usuniecia.

File: AI_NETCORE_API/Data/Infrastructure/PasswordValidation/Request/Abstract/IPasswordValidationRequest.cs [hunk 2]
```diff
@@ -0,0 +1,11 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Domain.Infrastructure.PasswordValidation.Request.Abstract
+{
+    public interface IPasswordValidationRequest
+    {
+        string PasswortToValidate { get; }
```
Comment (PawelBalcerek) [inline AI_NETCORE_API/Data/Infrastructure/PasswordValidation/Request/Abstract/IPasswordValidationRequest.cs]: PasswortToValidate -> PasswordToValidate

File: AI_NETCORE_API/AI_NETCORE_API/Controllers/ValuesController.cs (hunk 1 above)
Comment (CebulMaciej) [reply AI_NETCORE_API/AI_NETCORE_API/Controllers/ValuesController.cs:1]: Usuniete

File: AI_NETCORE_API/Data/Infrastructure/PasswordValidation/Request/Abstract/IPasswordValidationRequest.cs (hunk 2 above)
Comment (CebulMaciej) [reply AI_NETCORE_API/Data/Infrastructure/PasswordValidation/Request/Abstract/IPasswordValidationRequest.cs]: Zrobione

**Review by PawelBalcerek — APPROVED** [review]
Discussions resolved. Time to merge.

**Pull request merged**